### ISSUE TITLE
accessibility:  color contrast for <a> fails WCAG 2.0 minimum stds

### ISSUE THREAD

**Issue by weedySeaDragon** [scala/docs.scala-lang#941]
There are two main styles* that do not meet the minimum level of contrast between the foreground and background:
1. `<h1>` white foreground against a light blue background
2. `<a>` light blue foreground against a white background does not meet the WCAG 2.0 minimum standards for color contrast

_(* "main style" == heavily used in the website.  Perhaps there are some other styles, but these are the two that are prominent in the few pages I checked.)_

**_The definitions in style.css:_**
```

a {
  color: #23aad1;
  text-decoration: none;
  transition: all 350ms ease; }
  a:active, a:focus, a:hover {
    color: #DC322F;
    text-decoration: underline; }

...

.title-page {
  background: #5CC6E4;
  min-height: 200px; }
  .title-page h1 {
    font-size: 1.875rem;
    font-family: "Lato", sans-serif;
    padding-top: 40px;
    text-transform: uppercase;
    text-shadow: rgba(0, 43, 54, 0.1) 2px 2px 0;
    color: #fff; }

```

### References:
- Web Content Accessibility Guidelines (WCAG) 2.0 https://www.w3.org/TR/WCAG20/
- **a nice list of standards** (US 508, various international standards, WCAG etc), intro articles, etc:  https://webaim.org/resources/
-  **a free tool to check if a page meets minimum WCAG 2.0 (accessibility) standards**:  http://wave.webaim.org/ 
     _just enter the page URL and it runs checks and shows you what needs to be addressed_

#### Example of what the WAVE tool shows for the  [Getting Started](https://docs.scala-lang.org/getting-started.html) page:

<img width="897" alt="wcag-contrast-getting-started-wave-check" src="https://user-images.githubusercontent.com/673794/33499038-2541790a-d688-11e7-9cf3-856e57d3fe3d.png">

**Comment by weedySeaDragon** [comment]
In a https://github.com/scala/scala-lang/pull/891#issuecomment-386555832 comment  I encouraged @heathermiller to talk to the original site designer so that this can be addressed.  

This is about meeting _minimal, industry and globally accepted **standards**_ -- something important for all websites but especially critical for one so important as Scala so that the information can be shared with as many as possible.
